File: ESRGC.GIS.Domain/Model/Contract.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel.DataAnnotations;

namespace ESRGC.GIS.Domain.Model
{
    public class Contract
    {
        [ScaffoldColumn(false)]
        public int ContractId { get; set; }
        [MaxLength(50)]
        [Display(Name="Contract No.")]
        [Required]
        public string ContractNum { get; set; }
        [MaxLength(100)]
        [Display(Name="Contract Company")]
        public string ContractCompany { get; set; }
        [MaxLength(100)]
        [Display(Name="Company Name")]
        public string CompanyName { get; set; }
        [Display(Name = "Storm water")]
        public bool StormWater { get; set; }
        public bool Sanitary { get; set; }
        public bool Water { get; set; }
        public int? Status { get; set; }
    }
}

File: ESRGC.GIS.WebMap/Desktop/app/model/contract.js
/*
Author: Tu Hoang
ESRGC 2012
Desktop browser

Model: 
Contract 
contract.js
model representing data for contract
*/

ESRGC.Model.Contract = ESRGC.Class({
    CLASSNAME: 'ESRGC.Model.Contract',
    fields: [
        { name: 'ContractId', type: 'int' },
        { name: 'ContractNum', type: 'string' },
        { name: 'StormWater', type: 'boolean' },
        { name: 'Sanitary', type: 'boolean' },
        { name: 'Water', type: 'boolean' },
        { name: 'Status', type: 'int' }
    ]
}, ESRGC.Model.Base);